03-task-manager/starter/app.js

```diff
@@ -1 +1,34 @@
-console.log('Task Manager App')
+const express = require('express');
+const app = express();
+const tasks = require('./routes/tasks')
+const connectDB = require('./db/connect')
+require('dotenv').config()
+
+//middleware
+
+app.use(express.json())
+
+
+//routes
+app.get('/hello', (req, res) => {
+  res.send('Task Manager App')
+})
+
+app.use('/api/v1/tasks', tasks)
+
+
+const port = 3000;
+
+const start = async () => {
+  try {
+    await connectDB(process.env.MONGO_URI)
+    app.listen(port, console.log(`server is listening on port ${port}...`))
```

Comment:
You probably want to wrap ``console.log(`server is listening on port ${port}...`)`` in a callback function. As is, the console.log statement will be invoked immediately rather than after the app actually starts. See here for an example of how it's supposed to work: https://expressjs.com/en/starter/hello-world.html

Reply:
Will this be OK?
const start = async () => {
  try {
    await connectDB(process.env.MONGO_URI)
    app.listen(port, () => {
      console.log(`server is listening on port ${port}...`)
    })

Reply:
Yep, that works!